src/tests/integration/test_generic_db_operations.py

```diff
@@ -477,6 +477,68 @@ def test_sqlplot_pie(ip_with_dynamic_db, request, test_table_name_dict):
     assert type(out.result).__name__ in {"Axes", "AxesSubplot"}
 
 
+@pytest.mark.parametrize(
+    "ip_with_dynamic_db",
+    [
+        ("ip_with_postgreSQL"),
+        ("ip_with_duckDB"),
+        ("ip_with_Snowflake"),
+        ("ip_with_duckDB_native"),
+        ("ip_with_redshift"),
+        pytest.param(
+            "ip_with_SQLite",
+            marks=pytest.mark.xfail(reason="does not support schema"),
+        ),
+        pytest.param(
+            "ip_with_mariaDB",
+            marks=pytest.mark.xfail(reason="schema access denied"),
+        ),
+        pytest.param(
+            "ip_with_mySQL",
+            marks=pytest.mark.xfail(reason="schema access denied"),
+        ),
+        pytest.param(
+            "ip_with_MSSQL",
+            marks=pytest.mark.xfail(reason="sqlplot does not support SQL server"),
+        ),
+        pytest.param(
+            "ip_with_clickhouse",
+            marks=pytest.mark.xfail(
+                reason="Plotting from snippet not working in clickhouse"
+            ),
+        ),
+    ],
+)
```

Comment:
please open a new issue pointing to these tests so we tackle them later

Reply:
issue about the failing test cases : [here](https://github.com/ploomber/jupysql/issues/866)